config/typescript/global.d.ts

```diff
@@ -0,0 +1,7 @@
+declare namespace Intl {
```

Comment:
why isn't this added to the `i18n` package directly?

Reply:
Or is this part of the w3c spec?

Reply:
It seems to be referring to this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl